CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 Change Log
 ==========
+
+Version 2.0.5 *(2021-08-24)*
```

Comment:
Version also needs to be bumped in the `ApiConstants.SDK_VERSION` constant.

Reply:
Good catch, fixed now 👍